demos/instr.h

```diff
@@ -81,24 +81,22 @@ inline void JumpToAfterSpeculation() {
 #ifdef __i386__
 // Returns the original value of FS and sets the new value.
```

Comment:
Update comment for this and `ExchangeES`

Reply:
Obsolete.